db/deleteItem.go

```diff
@@ -0,0 +1,28 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+)
+
+func (d *dynamoDB) DeleteItem(listID *string, itemID *string) error {
+	input := &dynamodb.DeleteItemInput{
+		Key: map[string]*dynamodb.AttributeValue{
+			"ListId": {S: listID},
+			"Id":     {S: itemID},
+		},
+		TableName: aws.String(d.conf.TableNames.Items),
+	}
+
+	_, err := d.session.DeleteItem(input)
+
+	if err != nil {
```

Comment:
Could use the error switching that I put in the update PR here https://github.com/mount-joy/thelist-lambda/pull/9/files#r516787910

Reply:
I couldn't get this to work in the other PR but if I'm able to fix it there I'll update it here too

Reply:
https://play.golang.org/p/zqM53h68Cdz should help